### ISSUE TITLE
privoxy 3.0.31 conflicts with binutils

### ISSUE THREAD

**Issue by darkgeek** [termux/termux-packages#6369]
**Problem description**
Unable to update `privoxy` to `3.0.31`, because it conflicts with `binutils` on the file `/data/data/com.termux/files/usr/bin/aarch64-linux-android-strip`:

```shell
...
The following packages will be upgraded:
  privoxy
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/902 kB of archives.
After this operation, 1249 kB of additional disk space will be used.
(Reading database ... 71618 files and directories currently installed.)
Preparing to unpack .../privoxy_3.0.31_aarch64.deb ...
Unpacking privoxy (3.0.31) over (3.0.29) ...
dpkg: error processing archive /data/data/com.termux/cache/apt/archives/privoxy_3.0.31_aarch64.deb (--unpack):
 trying to overwrite '/data/data/com.termux/files/usr/bin/aarch64-linux-android-strip', which is also in package binutils 2.36.1
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
Errors were encountered while processing:
 /data/data/com.termux/cache/apt/archives/privoxy_3.0.31_aarch64.deb
E: Sub-process /data/data/com.termux/files/usr/bin/dpkg returned an error code (1)
```

**Steps to reproduce**
Install an old privoxy and try to update it by `pkg up -y`

**Expected behavior**
Update without that conflict.

**Additional information**
```shell
$ termux-info

Packages CPU architecture:                                                      
aarch64
Subscribed repositories:
# sources.list
deb https://mirrors.tuna.tsinghua.edu.cn/termux/termux-packages-24 stable main
# x11-repo (sources.list.d/x11.list)
deb https://mirrors.tuna.tsinghua.edu.cn/termux/x11-packages x11 main
# science-repo (sources.list.d/science.list)
deb https://mirrors.tuna.tsinghua.edu.cn/termux/science-packages-24 science stable
# game-repo (sources.list.d/game.list)
deb https://mirrors.tuna.tsinghua.edu.cn/termux/game-packages-24 games stable
Updatable packages:
privoxy/stable 3.0.31 aarch64 [upgradable from: 3.0.29]
Android version:
10
Kernel build information:
Linux localhost 4.9.227-gfd7b2c5f4652 #1 SMP PREEMPT Wed Jan 27 02:11:28 UTC 2021 aarch64 Android
Device manufacturer:
OnePlus
Device model:
ONEPLUS A6003
```

**Comment by xeffyr** [comment]
Interesting how `/data/data/com.termux/files/usr/bin/aarch64-linux-android-strip` appeared in privoxy deb file.

**Comment by xeffyr** [comment]
Fixed in `3.0.31-1` which will be available for installation soon.
